createContacts/index.ts

```diff
@@ -0,0 +1,48 @@
+import {AzureFunction, Context, HttpRequest} from '@azure/functions';
+import {contactRecord} from '../models/contact-Record';
+import {newContact} from '../models/new-Contact';
+import {getUserId, GuidClass, phoneValidator} from '../Common/Utils';
+
+const httpTrigger: AzureFunction = async function (
+  context: Context,
+  req: HttpRequest
+): Promise<void> {
+  let idValue = new GuidClass();
+  let idTemp = JSON.stringify(idValue.id);
+  const newContat_ = req.body as newContact;
+  const userTemp = getUserId();
+
+  let numStatus = await phoneValidator(req.body.phonenumber, userTemp);
+  if (numStatus) {
+    if (newContat_ && newContat_.surname && newContat_.name) {
+      context.bindings.outputDocument = {
+        // create a random ID
+        id: idTemp.slice(10, idTemp.length - 2),
+        userId: userTemp,
+        name: newContat_.name,
+        surname: newContat_.surname,
+        phonenumber: newContat_.phonenumber.replace(/[^0-9]/g, ''),
+      } as contactRecord;
+      context.res = {
+        status: 201,
+        body: newContat_,
+      };
+    } else {

```

Comment:
remember you can restructure this if/else to make it easier to read by simply inverting the if condition and dealing with the error case first